app/styles/partials/_info.scss

```diff
@@ -248,4 +248,58 @@ $packages: (
     width: 30px;
     order: 1;
   }
-}
\ No newline at end of file
+}
+
+.contributors img {
+  border-radius: 50%;
+  border: 2px solid #fff;
+  width: 30px;
+  margin-right: 5px;
+}
+
+/* Tooltip container */
+.tooltip {
+  position: relative;
+  display: inline-block;
+}
+
+/* Tooltip text */
+.tooltip .tooltiptext {
+  visibility: hidden;
+  width: 80px;
+  background-color: #000;
+  color: #fff;
+  text-align: center;
+  padding: 5px 0;
+  border-radius: 6px;
+  font-size: 12px;
+
+  /* Position the tooltip text */
+  position: absolute;
+  z-index: 1;
+  bottom: 125%;
+  left: 50%;
+  margin-left: -43px;
```

Comment:
More readable and less hardcoded solution would be:
`transform: translateX(calc(-50% + ($icon-size / 2))); // $icon-size = 30px`

Reply:
Nice! Surprisingly Sass doesn't translate the variable $icon-size in this expression (I get "$icon-size" in the output CSS). So eventually I used this expression:

```
transform: translateX(calc(-50% + (30px / 2))); // 30px = contributor avatar width
```

Reply:
Good catch, I forgot about one thing which is sass variable interpolation `#{$sass-variable}` (https://sass-lang.com/documentation/interpolation). This should work now:
`transform: translateX(calc(-50% + (#{$avatar-size} / 2)));`

Note: `$avatar-size` variable should be stored in `config.scss` file
I changed $icon-size to $avatar-size because $icon-size is existing already for some other elements